setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
Is it intentional that setup.py uses `#!/usr/bin/env python` while bin/todo uses `#!/usr/bin/python`?